linter.py

```diff
@@ -20,28 +20,12 @@
 class Flow(NodeLinter):
     """Provides an interface to flow."""
 
-    inline_settings = ('coverage', 'all')
-    syntax = (
-        'javascript',
-        'html',
-        'javascriptnext',
-        'javascript (babel)',
-        'javascript (jsx)',
-        'jsx-real'
-    )
     npm_name = 'flow-bin'
-    version_args = 'version --json'
-    version_re = r'"semver":\s*"(?P<version>\d+\.\d+\.\d+)"'
-    version_requirement = '>= 0.17.0'
 
     defaults = {
+        'selector': 'source.js',
         # Allow to bypass the 50 errors cap
-        'show-all-errors': True,
-        # Allow to use a specific executable
-        'flow-bin': None,
```

Comment:
`flow-bin` is not really necessary anymore, bc all linters can configure this by setting `executable`. This was really a high requested feature for SL4